code/modules/antagonists/cult/runes.dm

```diff
@@ -255,7 +255,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 			to_chat(M, span_warning("You need at least two invokers to convert [convertee]!"))
 		log_game("Offer rune failed - tried conversion with one invoker")
 		return FALSE
-	if(convertee.anti_magic_check(TRUE, TRUE, FALSE, 0)) //Not chargecost because it can be spammed
+	if(convertee.can_block_magic(MAGIC_RESISTANCE_HOLY|MAGIC_RESISTANCE_MIND, charge_cost = 0)) //No charge_cost because it can be spammed
```

Comment:
This was changed from magic and holy to holy and mind?

Reply:
Looks like I missed this when I reverted my holy resistance changes for cult back to regular magic resistance.  Fixed with https://github.com/tgstation/tgstation/pull/64124/commits/b81257cd0c4edb77329b209fd94f13d2e6973e72